changelog.md

```diff
@@ -19,6 +18,7 @@
 * Fix occasional sender's state is not completed and equal to the receiver's state when canceling the transfer
 * Unclutter transfer cleanup logs
 * Update moose tracker to v6.0.0
+* Windows ARM build
```

Comment:
Why?

Reply:
Because this release included Windows ARM build and as the changes were only in libdrop-build, I forgot to update the changelog then so I took the chance to fix this too now.

Reply:
Libdrop itself didn't have anything to do with the ARM build so it's fine to leave it unmention IMHO, it's just an artifact of libdrop-build

Reply:
Denis wanted it mentioned. In the no-context version it's mentioned.

Reply:
I can remove it, no problem

Reply:
Hmm.. I'd leave it here for history purpose, might be useful at some point :-?